api_feature.go

```diff
@@ -58,7 +58,7 @@ func (f *APIFeature) RegisterSteps(ctx *godog.ScenarioContext) {
 	ctx.Step(`^I PATCH "([^"]*)"$`, f.IPatch)
 	ctx.Step(`^I DELETE "([^"]*)"`, f.IDelete)
 	ctx.Step(`^the HTTP status code should be "([^"]*)"$`, f.TheHTTPStatusCodeShouldBe)
-	ctx.Step(`^the response header "([^"]*)" should be "([^"]*)"$`, f.TheResponseHeaderShouldBe)
+	ctx.Step(`^the response header "([^"]*)" should be "([^";]*)"$`, f.TheResponseHeaderShouldBe)
```

Comment:
Is this change intentional?

Reply:
Just saw this myself (when looking over), I will remove it